### PR TITLE
CIV-13272 Stop RPA for Judgment by Admission against Defendant LR/LIP

### PR DESCRIPTION
**Pull request by German-A-Amoros**
### JIRA link (if applicable) ###

[CIV-13272](https://tools.hmcts.net/jira/browse/CIV-13272)

### Change description ###

Stop RPA for Judgment by Admission against Defendant LR/LIP

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by hmcts-version1-ignacio — APPROVED** [review]

**Review by sjamihmcts — DISMISSED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/caseevents/RequestJudgementByAdmissionNonDivergentSpecCallbackHandler.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.reform.civil.handler.callback.camunda.caseevents;
```
Comment (Azam-Hmcts) [inline src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/caseevents/RequestJudgementByAdmissionNonDivergentSpecCallbackHandler.java:1]: Why you need this class ?

**Review by Azam-Hmcts — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/caseevents/RequestJudgementByAdmissionNonDivergentSpecCallbackHandler.java (hunk 1 above)
Comment (German-A-Amoros) [reply src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/caseevents/RequestJudgementByAdmissionNonDivergentSpecCallbackHandler.java:1]: I was told to create a handler for the new event on the last ticket, so I thought I had to do the same here.

Comment (Azam-Hmcts) [reply src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/caseevents/RequestJudgementByAdmissionNonDivergentSpecCallbackHandler.java:1]: I think you dont require it. This handler is doing something? Is it being called by camunda tasks?

File: src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RequestJudgementByAdmissionForSpecCuiCallbackHandler.java [hunk 2]
```diff
@@ -130,4 +147,13 @@ private String setUpBody() {
                 + "<br>This case will now proceed offline. Any updates will be sent by post.<br><br>"
         );
     }
+
+    private boolean isNonDivergent(CaseData caseData) {
+        return  MultiPartyScenario.isOneVOne(caseData)
+            || MultiPartyScenario.isTwoVOne(caseData)
```
Comment (Azam-Hmcts) [inline src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RequestJudgementByAdmissionForSpecCuiCallbackHandler.java:153]: If you are checking only  if "(featureToggleService.isJudgmentOnlineLive()
            && isNonDivergent(data)
            && data.isLRvLipOneVOne()
            && data.isPayImmediately())" at line 106 then why you need all this logic? you are checking here 1v2 scenarios also then only doing it for 1v1 scenario (LRvLip)

**Review by hmcts-version1-ignacio — APPROVED** [review]

**Review by sjamihmcts — APPROVED** [review]

**Review by krishnanuthalapati — APPROVED** [review]

**Pull request merged**